herbieapp/services/message_publisher/google_pub_sub_publisher.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from google.api_core.exceptions import AlreadyExists
+from google.cloud.pubsub_v1 import PublisherClient, SubscriberClient
+from herbie import settings
+from rest_framework.renderers import JSONRenderer
+
+from herbieapp.models.message_models_serializers import EntityUpdateMessage, EntityUpdateMessageSerializer, \
+    EntityDeleteMessage, EntityDeleteMessageMessageSerializer
+
+
+class GooglePubSubPublisher:
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self._logger = logging.getLogger(__name__)
+        self._publisher = PublisherClient()
+        self._subscriber = SubscriberClient()
+
+    def send_message(self, message):
+        serializer = self._get_serializer(message)
+
+        json_data = JSONRenderer().render(serializer.data)
+
+        topic_path = self._publisher.topic_path(settings.GCLOUD_PUBSUB_PROJECT_ID, 'customer')
```

Comment:
Is that 'customer' correct here?

Reply:
ooo nice catch Lars, it should be the type there